src/main/java/com/solvd/carina/swaglabs/components/ios/SortModal.java

```diff
@@ -0,0 +1,47 @@
+package com.solvd.carina.swaglabs.components.ios;
+
+import com.solvd.carina.swaglabs.components.common.SortModalBase;
+import com.zebrunner.carina.utils.mobile.IMobileUtils;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.locator.ExtendedFindBy;
+import org.openqa.selenium.WebDriver;
+
+public class SortModal extends SortModalBase implements IMobileUtils {
```

Comment:
A lot of action  in SortModal class, I think we can create enum class, and dynamic locator 
public enum SortType {

    PRICE_LOW_TO_HIGH(true, "Price (low to high)" ),
    PRICE_HIGH_TO_LOW(false, "Price (high to low)");
    }
    
    in class AndroidFilterComponent create below code
    
     @FindBy(xpath = "//android.widget.ScrollView[@content-desc='Selector container']//android.view.ViewGroup//android.widget.TextView[contains(@text, '%s')]")
    private ExtendedWebElement sortItemsBy;
    
     public void sortBy(SortType sortType) {
        sortItemsBy.format(sortType.getValue()).click();
    }